LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -299,6 +299,7 @@
     "web-platform-tests/html/semantics/forms": "import",
     "web-platform-tests/html/semantics/forms/textfieldselection/select-event.html": "skip",
     "web-platform-tests/html/semantics/forms/textfieldselection/textfieldselection-setRangeText.html": "skip",
+    "web-platform-tests/html/semantics/forms/the-select-element": "import",
     "web-platform-tests/html/semantics/forms/the-textarea-element": "import",
```

Comment:
These 2 lines are redundant with `"web-platform-tests/html/semantics/forms": "import",`

I wish the script didn't add them

Reply:
Ah I'll go ahead and drop it then.